pull_requests.md

```diff
@@ -51,6 +80,31 @@ best in __Refactoring: Improving the Design of Existing Code__:
 
 > “Any fool can write code that a computer can understand. Good programmers write code that humans can understand.“
 
+* Make in you comments concrete improvement suggestions with a clear rationale for the suggested change.
```

Comment:
How about
"Your comments should offer concrete suggestions for improvement with a clear..."